BrowserKit/Sources/TabDataStore/TabDataStore.swift

```diff
@@ -233,4 +239,33 @@ public actor DefaultTabDataStore: TabDataStore {
         let baseFilePath = filePrefix + windowID.uuidString
         return baseURL.appendingPathComponent(baseFilePath)
     }
+
+    // MARK: - Merge files
+
+    public func mergeWindowsData() async {
+        let windowIDs = fetchWindowDataUUIDs()
+
+        guard await UIDevice.current.userInterfaceIdiom != .pad,
+              windowIDs.count > 1,
+              let primaryID = windowIDs.first,
+              let primaryWindow = await fetchWindowData(uuid: primaryID)
+        else { return }
+
+        var newTabData = primaryWindow.tabData
+
+        for uuid in windowIDs {
+            if let nextWindow = await fetchWindowData(uuid: uuid) {
```

Comment:
```suggestion
            if uuid != primaryID, let nextWindow = await fetchWindowData(uuid: uuid) {
```
Given that you've already gotten the data from the `.first`, we want to do a loop on the tail of `windowIDs` rather than all, of it, no? Else you'd end up with the `.first`s data twice.

Reply:
well now that you fixed `newTabData` init, this comment makes no sense lol